hubblestack/extmods/grains/configgrains.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
Can you just add a docstring with sample config? This looks great otherwise.

Reply:
Done